### DISCUSSION TITLE
If you want to use Ultisnips...

### DISCUSSION THREAD

**Discussion by muzhancun** [ayamir/nvimdots#719] [Show and tell]
You can refer to [this link](https://github.com/SirVer/ultisnips/issues/1396#issuecomment-1407230253) if having problems to detect math zone in tex files. Just need to modify line 40 in [treesitter.lua](https://github.com/ayamir/nvimdots/blob/main/lua/modules/configs/editor/treesitter.lua), i.e. add "latex" to `additional_vim_regex_highlighting`

**Comment by charliie-dev** [comment]
gd tips, nice!
